README.md

```diff
@@ -1 +1,52 @@
-# homebrew-versions
\ No newline at end of file
+# Helm and Kubernetes-cli legacy versions
+
+Attempt to rebuild previous versions of clients needed for communication with existing older production infrastructure
+
+
+## Update with new formula
+
+When new version of Helm or Kubernetes-cli is released, brew will update its packages. To view all the previous versions of the tool, we can visit the brew histories of [Helm](https://github.com/Homebrew/homebrew-core/commits/7877b4bbcfa48028349a5cbed0d2f6ef5dbcc84b/Formula/helm.rb) and [Kubernetes-cli](https://github.com/Homebrew/homebrew-core/commits/674e098c3efce9196d91397d325c66d488aacc6b/Formula/kubernetes-cli.rb) to view list of all the previous versions.
+
+
+### Updating tap
+
+1. Create a file `helm@<x.x.x>.rb` or `Kubernetes-cli@<x.x.x>`
+
+    ``e.g:  helm@3.10.1.rb or kubernetes-cli@1.18.8``
+
+2. Copy the content of the formula
+    1. Visit formula history in homebrew
+
+    ![Tool version history link](./assets/latest_version_history.png)
+
+    2. Click on specific version bottle
+
+    ![Tool version bottle](./assets/package_version_bottle.png)
+
+    3. Copy the content of the formula from the bottle file to the newly created tap file.
+
+    ![Bottle file](./assets/bottlefile.png)
+
+3. Change the formula class name
+
+Class name of the formula should be changed to support installing specific version of the formula.
+
+***File name should be helm@<x.x.x>.rb or Kubernetes-cli@<x.x.x>
```

Comment:
Is this `AT<version` convention something that we have created? Or a convention documented by homebrew?

Reply:
That's the naming class naming convention need to be followed. Example existing formula in homebrew https://github.com/Homebrew/homebrew-core/blob/master/Formula/ansible%402.8.rb

Reply:
Is there a link to some homebrew documentation that explains why?

Reply:
https://docs.brew.sh/Versions.html

Reply:
Cool. A useful link to include in the docs :)

Reply:
Added to readme